### PR TITLE
Adds topic tooltip and refresh button

### PR DESCRIPTION
**Pull request by daniellemaxwell**
### Scope of changes

Adds a tooltip to the Project list page and to the Topics component of the Project Detail view. Also adds a refresh button to the Project List table.

Fixes SC-16371 & SC-16584

### Type of change

- [x] new feature
- [ ] bug fix
- [ ] documentation
- [ ] testing
- [ ] technical debt
- [ ] other (describe)

### Acceptance criteria

Screenshot 2023-05-03 at 3 22 44 PM

Screenshot 2023-05-03 at 2 53 14 PM

Ensign by Rotational Labs (13)

### Author checklist

- [ ] I have manually tested the change and/or added automation in the form of unit tests or integration tests
- [ ]  I have updated the dependencies list
- [ ]  I have recompiled and included new protocol buffers to reflect changes I made
- [ ]  I have added new test fixtures as needed to support added tests
- [x]  Check this box if a reviewer can merge this pull request after approval (leave it unchecked if you want to do it yourself)
- [x]  I have moved the associated Shortcut story to "Ready for Review"

### Reviewer(s) checklist

- [ ] Any new user-facing content that has been added for this PR has been QA'ed to ensure correct grammar, spelling, and understandability.
- [ ] Are there any TODOs in this PR that should be turned into stories?

### CODE REVIEW

File: web/beacon-app/src/features/projects/components/ProjectList.tsx [hunk 1]
```diff
@@ -27,10 +29,16 @@ function ProjectList() {
...
-      </Heading>
+      <div className="flex space-x-2 space-y-2">
+        <Heading as="h1" className="mb-4 mt-6 text-lg font-semibold">
+          Projects
+        </Heading>
+        <ProjectTooltip />
+      </div>
       <div className="flex justify-between rounded-lg bg-[#F7F9FB] px-3 py-2">
+        <div className="mt-2">
+          <RefreshIcon />
```
Comment (masskoder) [inline web/beacon-app/src/features/projects/components/ProjectList.tsx:40]: Maybe the refresh icon could be wrapped within a button? i'm wondering if it's not better to implement directly the logic ?

File: web/beacon-app/src/features/projects/components/ProjectTooltip.tsx [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+                  project requires an API key and secret. Secrets are known only to you. If you lose
+                  a secret, you lose the topic and data generated by the event stream permanently.
+                </Trans>
+              </p>
+            </Tooltip.Content>
+          </Tooltip.Portal>
+        </Tooltip.Root>
+      </Tooltip.Provider>
+    </>
+  );
```
Comment (masskoder) [inline web/beacon-app/src/features/projects/components/ProjectTooltip.tsx:35]: In the previous PR, i was talking about this kind of duplication. Maybe, creating in the `common` folder `HelpTooltip` component with content as props wouldn't be a better way to handle that?

File: web/beacon-app/src/features/projects/components/TopicToolTip.tsx [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+              align="start"
+            >
+              <p>
+                <Trans>
+                  Messages and events are sent to and read from specific topics. Services that are{' '}
+                  {''}
+                  <span className="font-bold">producers, write</span> data to topics. Services that
+                  are <span className="font-bold">consumers, read</span> data from topics. Topics
+                  are multi-subscriber, which means that a topic can have zero, one, or multiple
+                  consumers subscribing to that topic, with read access to the log.
```
Comment (masskoder) [inline web/beacon-app/src/features/projects/components/TopicToolTip.tsx:29]: Regarding this, you have two options for defining the type of content props (about the HelpTootip component). The first option is to set the type as ReactNode. Alternatively, you can use the children's props to wrap the content.

```
<HelpTooltip>
content with style 
</HelpTooltip>
```

**Review by masskoder — COMMENTED** [review]
Looks good, just a few minor required updates.

File: web/beacon-app/src/features/projects/components/ProjectList.tsx [hunk 4]
```diff
@@ -43,6 +44,11 @@ function ProjectList() {
         </HelpTooltip>
       </div>
       <div className="flex justify-between rounded-lg bg-[#F7F9FB] px-3 py-2">
+        <div className="mt-2">
+          <button disabled={isFetchingProjects} onClick={getProjects}>
```
Comment (daniellemaxwell) [inline web/beacon-app/src/features/projects/components/ProjectList.tsx:48]: In this video, you should be able to see where fetching highlights in the React Query dev tools when the refresh button is clicked. https://www.awesomescreenshot.com/video/17097918?key=f104eee28cf7ed39d50996057432c49c

File: web/beacon-app/src/features/projects/components/TopicToolTip.tsx (hunk 3 above)
Comment (daniellemaxwell) [reply web/beacon-app/src/features/projects/components/TopicToolTip.tsx]: Thanks for providing options here that were used in the other PR!

File: web/beacon-app/src/features/projects/components/ProjectList.tsx (hunk 1 above)
Comment (daniellemaxwell) [reply web/beacon-app/src/features/projects/components/ProjectList.tsx]: Thanks. I kept forgetting buttons yesterday for some reason.

File: web/beacon-app/src/features/projects/components/ProjectList.tsx [hunk 5]
```diff
@@ -15,7 +16,7 @@ function ProjectList() {
 
   const tenantID = tenants?.tenants[0]?.id;
 
-  const { projects } = useFetchTenantProjects(tenantID);
+  const { getProjects, isFetchingProjects, projects } = useFetchTenantProjects(tenantID);
 
   const [isOpenNewProjectModal, setIsOpenNewProjectModal] = useState<boolean>(false);
```
Comment (masskoder) [inline web/beacon-app/src/features/projects/components/ProjectList.tsx:22]:
```suggestion
const handleRefresh = () => {
   getProjects()
   }
```

File: web/beacon-app/src/features/projects/components/ProjectList.tsx (hunk 4 above)
Comment (masskoder) [inline web/beacon-app/src/features/projects/components/ProjectList.tsx]:
```suggestion
          <button disabled={isFetchingProjects} onClick={handleRefresh(}>
```

**Review by masskoder — APPROVED** [review]
Looks good,  i added a loader to the Table component. Great work

**Pull request merged**